CRM/Activity/StateMachine/Search.php

```diff
@@ -35,16 +35,10 @@ public function __construct($controller, $action = CRM_Core_Action::NONE) {
     $this->_pages = [];
 
     $this->_pages['CRM_Activity_Form_Search'] = NULL;
-    list($task, $result) = $this->taskName($controller, 'Search');
+    list($task) = $this->taskName($controller, 'Search');
     $this->_task = $task;
-
-    if (is_array($task)) {
```

Comment:
`$this->taskName` fixed to always return an array - it's not called from elsewhere